Classes/Widgets/AbstractDoughnutChartWidget.php

```diff
@@ -12,11 +12,10 @@
  *  This script is part of the TYPO3 project. The TYPO3 project is
  *  free software; you can redistribute it and/or modify
  *  it under the terms of the GNU General Public License as published by
- *  the Free Software Foundation; either version 2 of the License, or
- *  (at your option) any later version.
+ *  the Free Software Foundation; either any later version.
```

Comment:
Is this search&replace gone wrong? The official recommended text from the license v2 reads:

```
This program is free software; you can redistribute it and/or
modify it under the terms of the GNU General Public License
as published by the Free Software Foundation; either version 2
of the License, or (at your option) any later version.
```

Reply:
Also, this was definitely not the original recommended license notice -- and this software is (arguably) not "part of the TYPO3 project". Did we plagiarize this license notice from somewhere else?